### ISSUE TITLE
Example configuration has misnamed "shares" parameter

### ISSUE THREAD

**Issue by ryanwalls** [dencold/hasper#14]
Example config from README:
```
params:
  description: "David Hasselhoff loves telling you about Baywatch."
  cover: "images/cover.jpg"
  author: "david"
  logo: "images/logo.png"
  googleAnalyticsUserID: ""
  RSSLink: "http://feeds.feedburner.com/..."
  githubName: "thehoff"
  twitterName: "thehoff"
  hideHUGOSupport: false
  highlightjsTheme: "tomorrow-night-eighties"
  shares:
    twitter: false
    facebook: true
    google-plus: true
```

The actual parameter name for showing sharing options is "sharing".  See [share.html](layouts/partials/share.html).

**Comment by dencold** [comment]
Thanks for this, @ryanwalls. I am planning to completely revamp the README, per #3, this is a great stopgap to help anyone who wants to immediately start using the theme. If you want to submit this as a PR into my repo, I'll happily accept (right now you have a PR on your own repository).

Thanks again for contributing!!

**Comment by ryanwalls** [comment]
@dencold Thanks for catching that.  Don't know how I managed that with the PR.  Fixed now.

**Comment by dencold** [comment]
Fixed by @ryanwalls #15
